tools/clang/include/clang/SPIRV/SpirvBuilder.h

```diff
@@ -640,6 +640,11 @@ class SpirvBuilder {
   /// \brief Decorates the given target with Coherent
   void decorateCoherent(SpirvInstruction *target, SourceLocation);
 
+  /// \brief Decorates the given target with LinkageAttributes
```

Comment:
Could you please explain
- We have to set `targetInst` as `nullptr` when it is an imported or exported function
- We have to set `targetFunc` as `nullptr` when it is an imported or  exported global variable

in this comment?
Without this description, someone can misuse it.